ruby_programming/object_oriented_programming_basics/project_mastermind.md

```diff
@@ -9,7 +9,7 @@ Build a Mastermind game from the command line where you have 12 turns to guess t
 <div class="lesson-content__panel" markdown="1">
   1. Think about how you would set this problem up!
   2. Build the game assuming the computer randomly selects the secret colors and the human player must guess them.  Remember that you need to give the proper feedback on how good the guess was each turn!
-  3. Now refactor your code to allow the human player to choose whether he/she wants to be the creator of the secret code or the guesser.
+  3. Now refactor your code to allow the human player to choose whether they wants to be the creator of the secret code or the guesser.
```

Comment:
I think this should be changed to `they want to be`  instead of `they wants to be`